target/linux/qualcommax/ipq60xx/config-default

```diff
@@ -1,4 +1,5 @@
 CONFIG_IPQ_GCC_6018=y
+CONFIG_LEDS_PWM=y
```

Comment:
Until we see more devices using the PWM LED-s, you can just include `kmod-leds-pwm` for your device